docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -94,3 +94,42 @@ yarn add @material-ui/core
   -<Typography variant="srOnly">Create a user</Typography>
   +<Span>Create a user</Span>
   ```
+
+### Expansion Panel
+
+- Replace the `ExpansionPanel` name (and prefix) on the components with `Accordion`:
```

Comment:
A thought not directly related to the effort of this pull request. I wonder if we should explain why we make the changes. Breaking changes are painful, maybe if we provide a "why", it will improve the developers' experience.

Reply:
Added list of motivations for this.. Please take a look, not sure what is the best structure for this

Reply:
I would keep is very short. 

```suggestion
- Rename the `ExpansionPanel` components with `Accordion` to match the naming convention of the community:
```